x/twap/migrate.go

```diff
@@ -0,0 +1,16 @@
+package twap
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// MigrateExistingPools iterates through all pools and creates state entry for the twap module.
+func (k Keeper) MigrateExistingPools(ctx sdk.Context, latestPoolId uint64) error {
+	for i := 1; i <= int(latestPoolId); i++ {
+		err := k.afterCreatePool(ctx, uint64(i))
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
Thoughts on moving to listeners.go, keepers.go or api.go

Maybe i'm overly stubborn, with feeling bad about an extra file for this lol

Reply:
(this doesn't block PR merge)

Reply:
that I agree, but we're planning to delete this after upgrade in main no?
Feel like easier to manage them separately lmk what u think

Reply:
oh thats a good point, hrmm ok